packages/data-broker/src/services/dhis/pullers/DataElementsMetadataPuller.ts

```diff
@@ -38,25 +38,26 @@ export class DataElementsMetadataPuller {
     options: PullDataElementsOptions,
   ) => {
     const dataSourcesByDhisType = this.groupDataSourcesByDhisDataType(dataSources);
-    const metadata = [];
+    let metadata: DhisMetadataObject | DataElementMetadata[] = [];
```

Comment:
Why is the metadata defined as an object or an array here?

Reply:
Yeah you are right. It should be DataElementMetadata[]. Thanks!